### ISSUE TITLE
No focus indicator for people directory filters and search input

### ISSUE THREAD

**Issue by jeremyfelt** [washingtonstateuniversity/murrow.wsu.edu#130]
When tabbing to these areas at the top of the people directory, there is no indicator of focus. https://stage.murrow.wsu.edu/about/people/
